locales/base/translation.json

```diff
@@ -75,28 +75,9 @@
   "chooseFromLibrary": "Choose from Library",
   "takePhoto": "Take Photo",
   "removePhoto": "Remove Photo",
-  "accountSendLimit": "Account Send Limit",
```

Comment:
i decided to clean up the translations too. once these go to crowdin, they'll be removed from OTA translations but i think this is safe for older versions since the app does a "merge" of bundled and OTA translations.